pyston/test/test_recursion_limit.py

```diff
@@ -0,0 +1,43 @@
+import sys
```

Comment:
I think it makes sense to add this tests in the `Makefile` to the `_runtests` target.